servo/types/slo.py

```diff
@@ -17,12 +17,14 @@ class SloKeep(str, enum.Enum):
 class SloCondition(BaseModel):
     description: Optional[str] = None
     metric: str
+    slo_metric_minimum: float = 0.25
     threshold_multiplier: decimal.Decimal = decimal.Decimal(1)
     keep: SloKeep = SloKeep.below
     trigger_count: pydantic.conint(ge=1, multiple_of=1) = 1
     trigger_window: pydantic.conint(ge=1, multiple_of=1) = None
     threshold: Optional[decimal.Decimal]
     threshold_metric: Optional[str]
+    slo_threshold_minimum: Optional[float] = 0.25
```

Comment:
Don't need Optional if we're setting a default. Value should never be null
```suggestion
    slo_threshold_minimum: float = 0.25
```